st-embroider/2024/2024-06-18.md

```diff
@@ -0,0 +1,56 @@
+# 2024-06-18
+
+Note Taker: Chris Manson
+Time Keeper: 
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+- [x] Ed Falkner (EF)
+- [x] Chris Manson (CM)
+- [ ] Aaron Chambers (AC)
+- [x] Alex (A)
+- [ ] Preston Sego (PS)
+- [x] Simon Ihmig (SI)
+- [x] Katie Gengler (KG)
+- [ ] Peter Meehan (PM)
+- [ ] Krystan HuffMenne (KH)
+- [x] Peter Wagenet (PW)
+- [ ] James Davis (JD)
+- [x] Marine Dunstetter (MD)
+- [ ] Your name here (INITIALS)
+
+## Topics
+
+### macros config design for v2 apps <@ef4>
+
+- where do we want to put config for macros in v2 addons
+- classically we relied on it running code in the broccoli pipeline
+- we could put this in the explicit babel config 
+- this isn't the best design right now for authoring
+- the original RFC said that we could add build hooks to v2 addons
+- we could do that, but as we progressed we didn't think it was a great idea so we didn't implement it
+- CM: counter point - the ember-data issue isn't a good use case for the problem
```

Comment:
this was probably a misunderstanding of how EmberData's config works: since its just a method you import to set the macros config, it'll work anywhere. vite app blueprint just needs to make a choice as to where folks should import and invoke it.

Reply:
(also we took that approach since it made us agnostic to whether embroider ever added a config hook/file/pattern and means someone can use WarpDrive/EmberData sans embroider quite easily)